integrations/ragas/src/haystack_integrations/components/evaluators/ragas/metrics.py

```diff
@@ -170,12 +157,51 @@ def new(
         return cls(
             metric=metric,
             backend=backend,
+            input_validator=input_validator,
             input_parameters=input_parameters,
             input_converter=input_converter,
+            output_converter=output_converter if output_converter is not None else OutputConverters.default,
             init_parameters=init_parameters,
         )
 
 
+class InputValidators:
```

Comment:
This should be renamed according to the above. Also, the `pydoc` config needs to be updated.